default.nix

```diff
@@ -1,29 +1,21 @@
-{ mkDerivation, aeson, base, bytestring, case-insensitive
-, containers, data-default, exceptions, ghcjs-dom, http-api-data
-, http-media, jsaddle, mtl, network-uri, reflex, reflex-dom
-, reflex-dom-core, safe, scientific, servant, servant-auth, stdenv
+{ mkDerivation, base, bytestring, case-insensitive, containers
+, data-default, exceptions, ghcjs-dom, http-api-data, http-media
+, jsaddle, jsaddle-dom, mtl, network-uri, reflex, reflex-dom
+, reflex-dom-core, safe, servant, servant-auth, stdenv
 , string-conversions, text, transformers
 }:
 mkDerivation {
   pname = "servant-reflex";
   version = "0.3.4";
-  src = builtins.filterSource
-        (path: type:
-        baseNameOf path != "result"
-        && baseNameOf path != "nix" 
-        ) ./.;
-  configureFlags = [ "-fexample" ];
```

Comment:
Is this intentionally part of the PR? I build the example as part of the CI process.

Reply:
no I just did the cabal2nix thing

Reply:
readded it